src/Header.svelte

```diff
@@ -0,0 +1,100 @@
+<script>
+  import { tasks } from "./stores";
+
+  // buttons are so 2000's, let's use emojis instead
+  import EmojiButton from "./EmojiButton.svelte";
+
+  // tooltips are helpful
+  import ToolTip from "./ToolTip.svelte";
+
+  let taskDescription = "";
+
+  // string to hold the title
+  $: title = selectRandomTitle();
+
+  // function to select a random title
+  function selectRandomTitle() {
+    const titles = [
+      "To-Do List",
+      "Aufgabenliste",
+      "Lista de Pendientes",
+      "Liste de Choses à Faire",
+      "Lista di Cose da Fare",
+      "Lista de Afazeres",
+      "Список Дел",
+      "待办事项",
+      "タスク一覧",
+      "רשימת המטלות",
+      "🍈🍈🍈", //get it, honeydew? honey do? i'm hilarious...
+    ];
+    const index = Math.floor(Math.random() * titles.length);
+    return titles[index];
+  }
+
+  // function to add task if users presses enter, instead of clicking the button
+  const onKeyDown = (event) => {
+    if (event.key === "Enter") {
+      event.preventDefault();
+      addTask();
+    }
+  };
+
+  // function to add a new task to the array
+  function addTask() {
+    if (taskDescription !== "") {
+      tasks.update((values) => [
+        ...values,
+        { done: false, text: taskDescription, deleted: false },
+      ]);
+    }
+    taskDescription = "";
+  }
+</script>
+
+<header>
+  <h1>{title}</h1>
+  <div>
+    <ToolTip
+      title="Type something that needs to get done, then press Enter or click ✔️ below to add it to the list."
+    >
+      <input
+        id="description"
+        name="description"
+        type="text"
+        bind:value={taskDescription}
+        placeholder="What needs to be done?"
+        on:keydown={onKeyDown}
```

Comment:
This was `on:keypress`, but I swapped it out for `on:keydown` because the [`keypress` event is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event).